scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -377,13 +392,15 @@ def unpacker(res):
         message = 'The input contains nan values'
         with pytest.raises(ValueError, match=message):
             hypotest(*data, axis=axis, nan_policy=nan_policy, *args, **kwds)
-
+    elif (nan_policy == 'omit' and data_generator in {"all_nans", "mixed"}
+          and hypotest not in too_small_special_case_funcs):
+        with pytest.warns(SmallSampleWarning, match=too_small_nd_omit):
+            hypotest(*data, axis=axis, nan_policy=nan_policy, *args, **kwds)
```

Comment:
This is still an open TODO, or am I misunderstanding your comment?

Reply:
> This needs to be moved into the `else` block, among other things.

Can you comment about the status of this? And if follow-ups are necessary, open a new issue?

Reply:
Not sure why this doesn't show up in the main thread.

Reply:
Oh, weird. Probably doesn't show up because this refers to old code. At the time you commented originally, it still needed work, but it has since been rewritten. So no, no follow-ups here.